ga4gh/cli.py

```diff
@@ -133,7 +134,7 @@ def _jsonOutput(self, gaObjects):
         line.
         """
         for gaObject in gaObjects:
-            print(gaObject.toJsonString())
+            print(protocol.toJson(gaObject, indent=None))
```

Comment:
Probably, actually yes. toJson defaulted to 2 space pretty-printing originally, because that's what had been put in by Ian's initial conversion of this, but now it should probably be converted to default to no indentation.